### ISSUE TITLE
[external CONTACT]

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#6305]
![image](https://github.com/ArctosDB/arctos/assets/14808196/9e491614-b645-44e4-bbe9-0e3236cbdbae)

<html>
<body>
<div class="aju" style="cursor: default; align-items: center; display: flex; float: none; height: 80px; padding: 0px 16px; min-width: 40px; color: rgb(34, 34, 34); font-family: &quot;Google Sans&quot;, Roboto, RobotoDraft, Helvetica, Arial, sans-serif; font-size: medium; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="aCi" style="position: relative;"><img id=":p9_186-e" name=":p9" src="https://lh3.googleusercontent.com/cm/AOLgnvtaz6my0dIoDYP_vVeWVciUg-4Zg0NSHPSJMxZAGhBJkwnMFwRmvSFf2aymAfFs=s40-p" class="ajn" jid="arctoscontact@googlegroups.com" data-hovercard-id="arctoscontact@googlegroups.com" data-name="'Ramirez, Desiree' via ArctosContact" aria-hidden="true" style="width: 40px; height: 40px; border-radius: 50%; display: block; background-color: rgb(204, 204, 204);"></div></div><div class="gs" style="margin: 0px; padding: 0px 0px 20px; width: 1052px; color: rgb(34, 34, 34); font-family: &quot;Google Sans&quot;, Roboto, RobotoDraft, Helvetica, Arial, sans-serif; font-size: medium; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="gE iv gt" style="font-size: 0.875rem; cursor: auto; padding: 20px 0px 0px;">

'Ramirez, Desiree' via ArctosContact | 'Ramirez, Desiree' via ArctosContact | 3:31 PM (31 minutes ago) |   |  
-- | -- | -- | -- | --
'Ramirez, Desiree' via ArctosContact
to arctoscontact@googlegroups.com | to arctoscontact@googlegroups.com
to arctoscontact@googlegroups.com

</div><div id=":2qz"><div class="qQVYZb"></div><div class="utdU2e"></div><div class="lQs8Hd" jsaction="SN3rtf:rcuQ6b" jscontroller="i3Ohde"></div><div class="btm"></div></div><div class=""><div class="aHl" style="margin-left: -38px;"></div><div id=":44g" tabindex="-1"></div><div id=":2x0" class="ii gt" jslog="20277; u014N:xr6bB; 1:WyIjdGhyZWFkLWY6MTc2NjE3ODUxNzk5ODA5MTUwMyIsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsW11d; 4:WyIjbXNnLWY6MTc2NjE3ODUxNzk5ODA5MTUwMyIsbnVsbCxbXV0." style="direction: ltr; margin: 8px 0px 0px; padding: 0px; position: relative; font-size: 0.875rem;"><div id=":2vz" class="a3s aiL msg3853422766023914871" style="font: small / 1.5 Arial, Helvetica, sans-serif; overflow: hidden;"><div class="adM"></div><div dir="ltr"><div class="adM"></div><div style="font-family: Calibri, Arial, Helvetica, sans-serif; font-size: 12pt; color: rgb(0, 0, 0);">Hello,</div><div style="font-family: Calibri, Arial, Helvetica, sans-serif; font-size: 12pt; color: rgb(0, 0, 0);"><br></div><div style="font-family: Calibri, Arial, Helvetica, sans-serif; font-size: 12pt; color: rgb(0, 0, 0);">I am trying to get on Arctos but keep getting the following error, is the website down?</div><div style="font-family: Calibri, Arial, Helvetica, sans-serif; font-size: 12pt; color: rgb(0, 0, 0);"><br></div><div style="font-family: Calibri, Arial, Helvetica, sans-serif; font-size: 12pt; color: rgb(0, 0, 0);"><img id="m_3853422766023914871imageSelected0" width="287" height="273" size="20324" src="https://mail.google.com/mail/u/0?ui=2&amp;ik=218e3c31f5&amp;attid=0.1&amp;permmsgid=msg-f:1766178517998091503&amp;th=1882ba012733bcef&amp;view=fimg&amp;fur=ip&amp;sz=s0-l75-ft&amp;attbid=ANGjdJ-jsywUTXF33CkimXGOCL3ujaYKYip1Rf3nwnKDcbk_dmdqBEPYLRw91rbcA1wLh5z2y6Fxfkm9cY_ydqSmNL3Z1SsTk39bkc8pUergCDF5OEA07O8fLwzDrfQ&amp;disp=emb" data-image-whitelisted="" class="CToWUd a6T" data-bit="iit" tabindex="0" style="cursor: pointer; outline: 0px; max-width: 100%; width: 287.5px; height: 273px;"><br></div><div style="font-family: Calibri, Arial, Helvetica, sans-serif; font-size: 12pt; color: rgb(0, 0, 0);"><br></div><div style="font-family: Calibri, Arial, Helvetica, sans-serif; font-size: 12pt; color: rgb(0, 0, 0);">Thanks!</div></div></div></div></div></div>
</body>
</html>

**Comment by dustymc** [comment]
Doesn't seem like anything Arctos or TACC might have done, and none of us can replicate. Chris guesses it could possibly be related to the recent DNS changes (so try restarting everything).

**Comment by ewommack** [comment]
Hi @dustymc - so a restart on what specifically?
Clean out caches, restart modems, restart their handshakes?

**Comment by dustymc** [comment]
>restart on what specifically?

Everything==I have no way to tell. It could be anything from something at their ISP to their browser, and it could not be the only thing we (who have very little actual information) can imagine it to be at all.

**Comment by ewommack** [comment]
Thanks @dustymc. I'll see what I can do.

**Comment by ewommack** [comment]
From Desiree:
Hello Beth,

Thank you for the fast response. I am on a federal computer but that has not been an issue in the past. We have many specimens on loan with the University of Alaska Fairbanks that I access via Arctos on a regular basis. However, I did just try to access the website via my cell phone and I was able to get onto Arctos. I will conduct my search via another (non-federal) computer.

Thanks again!

**Comment by ewommack** [comment]
And final response to Desiree below. Since it seems it appears they can use a work around I'll close this issue:
Hello Desiree,

Glad there is a quick solution for you. We have gotten a similar notice from other federal agents in Alaska on problems with accessing Arctos recently. 

In case you do need to use your federal computer here is what we have been able to figure out so far (also duplicated on the Github Issue I shared with you in the previous email):
Both our programmer and TACC which houses our database are unable to replicate your problem. The suggestion is that there may be a problem from the federal side with the DNS changes that Arctos just renewed. The recommendation is to try restarting everything.

Thank you,
Beth
